.Lib9c.Tests/Helper/PetHelperTest.cs

```diff
@@ -0,0 +1,107 @@
+namespace Lib9c.Tests.Helper
+{
+    using System;
+    using System.Linq;
+    using Nekoyume.Helper;
+    using Nekoyume.TableData.Pet;
+    using Xunit;
+
+    public class PetHelperTest
+    {
+        private readonly PetCostSheet _petCostSheet;
+
+        public PetHelperTest()
+        {
+            Assert.True(TableSheetsImporter.TryGetCsv(nameof(PetCostSheet), out var csv));
+            _petCostSheet = new PetCostSheet();
+            _petCostSheet.Set(csv);
+        }
+
+        [Fact]
+        public void CalculateEnhancementCost()
+        {
+            var row = _petCostSheet.First!;
+            var petId = row.PetId;
+            var cost = row.Cost.FirstOrDefault(cost =>
+                cost.NcgQuantity > 0 ||
+                cost.SoulStoneQuantity > 0);
+            Assert.NotNull(cost);
+            PetHelper.CalculateEnhancementCost(
+                _petCostSheet,
+                petId,
+                0,
+                cost.Level);
+        }
+
+        [Fact]
+        public void CalculateEnhancementCost_Throw_ArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+                PetHelper.CalculateEnhancementCost(
+                    null,
+                    0,
+                    0,
+                    0));
+        }
+
+        [Fact]
+        public void CalculateEnhancementCost_Throw_ArgumentException()
+        {
+            var row = _petCostSheet.First!;
+            var petId = row.PetId;
+            Assert.Throws<ArgumentException>(() =>
+                PetHelper.CalculateEnhancementCost(
+                    _petCostSheet,
+                    -1,
+                    0,
+                    1));
```

Comment:
How about moving these args to `inlineData` of test?
I cannot get what `-1, 0, 1` means until look inside of `CalculateEnhancement` function.